torch/ao/quantization/fx/convert.py

```diff
@@ -174,7 +195,33 @@ def _replace_observer_with_quantize_dequantize_node(
         quantize_op = "to"
         qparams = {"_dtype_": dtype}
 
-    # 2. replace observer node with quant - dequant node
+    # 2. insert choose_qparams op and update the params list for the is_decomposed and
+    # is_dynamic case
+    if is_decomposed:
+        if is_dynamic:
```

Comment:
now this case matches, but the reader is left with the question of "what about non-decomposed" or "non-dynamic".  Can we add some comments on why we don't have four cases here?